optimum/onnxruntime/modeling_ort.py

```diff
@@ -1230,6 +1253,12 @@ def forward(
         token_type_ids: Optional[Union[torch.Tensor, np.ndarray]] = None,
         **kwargs,
     ):
+        # Handle return_dict from kwargs with default value False
+        return_dict = kwargs.pop("return_dict", False)
```

Comment:
Is there any reason why `return_dict` is not added as an optional argument in the `forward` function instead so that it can be taken directly instead of retrieved from `kwargs`? Is it because the arguments must be `torch.Tensor | np.ndarray | None`?

```suggestion
        return_dict: bool = False,
        **kwargs,
    ):
```